src/pages/en/guides/rss.mdx

```diff
@@ -126,6 +128,13 @@ export async function get(context) {
 }
 ```
 
+:::note[Using an older version?]
+In versions `@astrojs/rss` before v2.1.0, pass your glob result straight to `items` without the `pagesGlobToRssItems` wrapper:
```

Comment:
I think you accidentally changed the order here.
```suggestion
In `@astrojs/rss` versions before v2.1.0, pass your glob result straight to `items` without the `pagesGlobToRssItems` wrapper:
```

Reply:
Above, we include the `()` for `pagesGlobToRssItems()`. We should choose one and be consistent throughout.  :smile:

Reply:
Good catch — thank you! I meant to include an “of” in between there. Updated with the function parentheses too:

```suggestion
In versions of `@astrojs/rss` before v2.1.0, pass your glob result straight to `items` without the `pagesGlobToRssItems()` wrapper:
```